src/offchainapi/payment_logic.py

```diff
@@ -454,28 +454,14 @@ def check_new_payment(self, new_payment):
         # TODO: catch exceptions into Payment errors
 
         try:
-            sub_send = LibraAddress.from_encoded_str(new_payment.sender.address)
-            sub_revr = LibraAddress.from_encoded_str(new_payment.receiver.address)
+            send_addr = LibraAddress.from_encoded_str(new_payment.sender.address)
+            receiver_addr = LibraAddress.from_encoded_str(new_payment.receiver.address)
         except LibraAddressError as e:
             raise PaymentLogicError(
                 OffChainErrorCode.payment_invalid_libra_address,
                 str(e)
             )
 
-        # TODO: TEST and fix these
```

Comment:
This is really the core of the change besides changing variable names right? If so it is fine if you really feel that subaddresses may be optional in some use cases.

Reply:
will remove this line

Reply:
that was actually removed already